docs/analytics/apis/user-privacy-api.md

```diff
@@ -70,7 +76,7 @@ The body parameter is required. It's the deletion request object listing the `us
 
     ```bash
     curl --location --request POST 'https://amplitude.com/api/2/deletions/users' \
-    --header 'Authorization: Basic {{api-key}}:{{secret-key}} \
+    --header 'Authorization: Basic {{api-key}}:{{secret-key}} \ # API Key:Secret Key must be Base64-encoded
```

Comment:
I love the `# API Key:Secret Key must be Base64-encoded` comment!

How about putting something like:
`--header 'Authorization: Basic {{base64-encoded-credentials}} \ # "{{api-key}}:{{secret-key}}" pair must be Base64-encoded`

Also, let's make similar changes in other places, which mention `Authorization: Basic {{api-key}}:{{secret-key}}`

Reply:
This is my fault -- I wrongly assumed that anyone working with the API would know that basic auth requires base64 encoding of creds. Forgive me. SO, this needed to be added to the Basic Auth reusable snippet we use for every API that uses Basic Auth see #430  for that work.